doc/logging/logger.md

```diff
@@ -0,0 +1,7 @@
+# Bluesky logger
+
+To invoke the bluesky logger, import as `from ibex_bluesky_core.logger import logger` and use it at the desired level:
+`logger.blueskylogger.warning("Message to be logged")`
+The logger utilizes a `TimedRotatingFileHandler` defined in the `logging.conf` file that rolls over the log at midnight.
+
+The default logging level is defined at `INFO`.
```

Comment:
Nitpicking a bit here sorry- I think we should provide some more info on how to change the log level as this is one of the acceptance criteria. e.g Where do I find info on what different log levels there are? And where do I make this change?

Reply:
The logging levels are sort of python standard - DEBUG, INFO, WARNING etc. And we invoke them as --> mylogger.warning("message"), mylogger.error("another message"). I think it might be overkill to describe the python standard logging levels and calls here. 
As described here, the log handler is defined in logging.conf and that's where the default logging level is set to INFO for the handler

Reply:
Okay cheers thats all fine then, merging now